content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Getting Started with AWS
+kind: documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/aws-monitoring/'
+      tag: 'Blog'
+      text: 'Key metrics for AWS monitoring'
+    - link: 'https://www.datadoghq.com/blog/aws-1-click-integration/'
+      tag: 'Blog'
+      text: 'Introducing our AWS 1-click integration'
+    - link: agent/faq/why-should-i-install-the-agent-on-my-cloud-instances/
+      tag: 'FAQ'
+      text: 'Why should I install the Datadog agent on my cloud instances?'
+    - link: 'integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose/??tab=cloudformation'
+      tag: 'Documentation'
+      text: 'Use CloudWatch metric streams with Kinesis Data Firehose'
+      
+---
+
+## Overview
+
+This guide provides an overview of the process for linking an AWS account with Datadog using Datadog's CloudFormation template. 
+
+At a high level, this involves creating a read-only IAM role and associated policy for Datadog's AWS account and deploying the [Datadog Forwarder][1] Lambda function. The forwarder then simplifies the process of sending logs, custom metrics, and traces to Datadog. Using the CloudFormation template provides all the tools needed to send this data to your Datadog account, and Datadog maintains the CloudFormation template to provide the latest functionality.
+
+This process can be repeated for as many AWS accounts as necessary, and you can also use the [API][2] or [Terraform][3] to set up multiple accounts at once.
+
+## Prerequisites
+
+Before getting started, ensure you have the following prerequisites:
+
+1. [AWS][4] account
+2. A Datadog account and [API key][5]. If you need a Datadog account, [sign up for a free trial][6].
+
+## Setup
+
+1. From the AWS tile on the [Integrations Page][7] in your Datadog account. Select the CloudFormation Template option:
+{{< img src="getting_started/integrations/integration-tile-setup.png" alt="Datadog integrations AWS tile">}}
+
+2. This opens the AWS Console and loads the CloudFormation stack. For this demonstration, the API key has been entered directly into the `DdApiKey` field, but you can also store this value in [Secrets Manager][8] and specify the ARN of the secret:
+{{< img src="getting_started/integrations/cloudformation-options-1.png" alt="First part of CloudFormation options for creating Datadog stacks">}}
+
+3. Ensure that the the correct [Datadog site][9] parameter is used for the `DdSite` parameter to match the region of your Datadog account. Check the required boxes from AWS and click `Create stack`:
```

Comment:
```suggestion
3. Ensure that the correct [Datadog site][9] parameter is used for the `DdSite` parameter to match the region of your Datadog account. Check the required boxes from AWS and click `Create stack`:
```
I am confused by this one, I don't see where you would put the DD site in this screenshot